src/chatdbg/__main__.py

```diff
@@ -6,6 +6,8 @@
 
 sys.path.insert(0, os.path.abspath(the_path))
 
-from . import chatdbg
+from .chatdbg import *
```

Comment:
I am really not a fan of `import *`. It can shadow existing imports/variables silently and heavily messes with any static analysis tool (not that we have any currently). I'll just fix it after this merges.